### PR TITLE
Javascript3/week1

### PR DESCRIPTION
**Pull request by narendra28051991**

### CODE REVIEW

File: javascript/javascript3/week1/giphy_searcher/js/app.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const text = document.getElementById('text')
```
Comment (naresh699) [inline javascript/javascript3/week1/giphy_searcher/js/app.js:1]: avoid name of var which are reserve keys in JS.

File: javascript/javascript3/week1/happy_project/assets/vendors/bootstrap/bootstrap.affix.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
+/* ========================================================================
+ * Bootstrap: affix.js v3.3.6
+ * http://getbootstrap.com/javascript/#affix
+ * ========================================================================
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ * ======================================================================== */
+
+
++function ($) {
```
Comment (naresh699) [inline javascript/javascript3/week1/happy_project/assets/vendors/bootstrap/bootstrap.affix.js:10]: do u need this + sign

File: javascript/javascript3/week1/happy_project/assets/vendors/bootstrap/bootstrap.affix.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  'use strict';
+
+  // AFFIX CLASS DEFINITION
+  // ======================
+
+  var Affix = function (element, options) {
```
Comment (naresh699) [inline javascript/javascript3/week1/happy_project/assets/vendors/bootstrap/bootstrap.affix.js:16]: do we need var name in uppercase, usually write in lowecase

**Review by naresh699 — APPROVED** [review]
you can merge code

File: javascript/javascript3/week1/giphy_searcher/js/app.js (hunk 1 above)
Comment (narendra28051991) [reply javascript/javascript3/week1/giphy_searcher/js/app.js:1]: Sur Naresh. I agree with you. I will change it.

File: javascript/javascript3/week1/happy_project/assets/vendors/bootstrap/bootstrap.affix.js (hunk 2 above)
Comment (narendra28051991) [reply javascript/javascript3/week1/happy_project/assets/vendors/bootstrap/bootstrap.affix.js:10]: Happy_project is created for us by default. We are just responsible for the scripts. I just implemented the functionality with the API.

**Pull request merged**